source/includes/project.java

```diff
@@ -0,0 +1,85 @@
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.ServerApi;
+import com.mongodb.ServerApiVersion;
+import com.mongodb.reactivestreams.client.MongoClients;
+import com.mongodb.reactivestreams.client.FindPublisher;
+import com.mongodb.reactivestreams.client.MongoClient;
+import com.mongodb.reactivestreams.client.MongoCollection;
+import com.mongodb.reactivestreams.client.MongoDatabase;
+import org.bson.BsonDocument;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.mongodb.client.model.Filters.eq;
+import static com.mongodb.client.model.Projections.*;
+
+class QueryDatabase {
+    public static void main(String[] args) {
+        // Replace the placeholder with your Atlas connection string
+        String uri = "<connection string>";
+
+        // Construct a ServerApi instance using the ServerApi.builder() method
+        ServerApi serverApi = ServerApi.builder()
+                .version(ServerApiVersion.V1)
+                .build();
+
+
+        MongoClientSettings settings = MongoClientSettings.builder()
+                .applyConnectionString(new ConnectionString(uri))
+                .serverApi(serverApi)
+                .build();
+
+
+        // Create a new client and connect to the server
+        try (MongoClient mongoClient = MongoClients.create(settings))
+        {
+           MongoDatabase database = mongoClient.getDatabase("sample_restaurants");
+           MongoCollection<Document> restaurants = database.getCollection("restaurants");
+
+//  start-project-include
+FindPublisher<Document> findProjectionPublisher = restaurants.find(
```

Comment:
S: I think the indentation is off beginning here

Reply:
Hey,

I originally indented it properly, but then the code snippets in the actual docs looked weird (were indented way too much in), so I changed it back. Do you have a suggestion to how to avoid that?

Reply:
Resolved by talking to Rea over slack